### PR TITLE
Typo its -> it's

### PR DESCRIPTION
**Pull request by NilsIrl**

### CODE REVIEW

**Comment by ms-jpq** [comment]
can I keep this pull request open because its literally the funniest thing i've had happen to me?

:D :D :D 

thx <3 love u

File: docs/PERF.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ In general, there are two ways to make a program faster.
 
 The storage & query engine in `coq.nvim` is written in SQL.
 
-SQLite comes natively with python, and its about as battle tested as anything out there.
+SQLite comes natively with python, and it's about as battle tested as anything out there.
```
Comment (cclauss) [inline docs/PERF.md:41]:
```suggestion
SQLite comes natively with Python, and it's about as battle tested as anything out there.
```
python is a 🐍 while Python is a programming language. ;-)

**Comment by Mte90** [comment]
There is also a typo on https://github.com/ms-jpq/coq_nvim/blob/968852a65af5ccb4e0cd6cc0c13aab22b29c8a95/coq/client.py#L89

**Comment by ms-jpq** [comment]
...

**Comment by Mte90** [comment]
> ...

We are developers not grammar nazi, happens 😂

**Comment by NilsIrl** [comment]
Fixed by 335f9fdede41010ef5a35eba56b6d722b308e3f4

**Pull request closed without merging**